PYME/Acquire/SpoolController.py

```diff
@@ -265,41 +297,49 @@ def _get_queue_name(self, fn, pcs=False):
         return self._sep.join([self.dirname.rstrip(self._sep), fn + ext])
 
 
-    def StartSpooling(self, fn=None, stack=None, compLevel=None, zDwellTime=None, doPreflightCheck=True, maxFrames=sys.maxsize,
-                      pzf_compression_settings=None, cluster_h5 = None):
-        """Start spooling
+    def StartSpooling(self, filename=None, max_frames=sys.maxsize, 
+                      do_preflight_check=True, **kwargs):
         """
+
+        Parameters
+        ----------
+        filename : str, optional
+            filename can be hardcoded here, otherwise differs to the seriesName
+            property which will create one if need-be.
+        max_frames : int, optional
```

Comment:
variable name changes violate contributing guidelines and change API.

Reply:
We are changing the entire signature - while we do it is it not a good idea to match what you have in `SpoolControllerWrapper.start_spooling`?